dmsite/venv/src/ProfileComponent.js

```diff
@@ -1,29 +1,52 @@
 import React, { Component } from 'react';
 import { withRouter } from 'react-router-dom';
-import { Container, Row, Col } from 'reactstrap';
 import FilterColumn from './DataColumn/FilterColumn';
 import DisplayData from './DataColumn/DisplayData';
 import DisplayColumn from './VisualColumn/DisplayColumn';
 import Results from './ClassificationColumn/Results';
+import { Container, Card, Button,CardTitle, CardBody, InputGroup, InputGroupText, InputGroupAddon, Input} from 'reactstrap';
 
 class Profile extends Component{
     render(){
         return (
-            <Container fluid>
-                <h1>Campaign: Current Campaign</h1>
-                <p> This campaign organizes Data1, Data2, Data3 </p>
-                <Row>
-                    <Col md="3">
-                        <FilterColumn name={'Datasets'} />
-                        <DisplayData name={'File'}/>
-                    </Col>
-                    <Col md="5">
-                        <Results name={"Classifications"} />
-                    </Col>
-                    <Col md="4">
-                        <DisplayColumn name={'Display Actions'} />
-                    </Col>
-                </Row>
+            <Container fluid style={{ justifyContent: 'center' }}>
+              <Card style={{ width: '60%', justifyContent: 'center', alignSelf: "center" }}>
+                <CardTitle className="pl-4 pt-4 pb-0">UserProfile</CardTitle>
+                <CardBody style={{ width: '100%', justifyContent: 'space-between' }}>
+                  <InputGroup>
+                    <InputGroupAddon addonType="prepend">
+                      <InputGroupText>@Username</InputGroupText>
+                    </InputGroupAddon>
+                    <Input />
+                  </InputGroup>
+                  <br />
+                  <InputGroup>
+                    <InputGroupAddon addonType="prepend">
+                      <InputGroupText>@Email!</InputGroupText>
+                    </InputGroupAddon>
+                    <Input />
+                  </InputGroup>
+                  <br />
+                  <InputGroup>
+                    <InputGroupAddon addonType="prepend">
+                      <InputGroupText>@Password</InputGroupText>
+                    </InputGroupAddon>
+                    <Input />
+                  </InputGroup>
+                  <br />
+                  <InputGroup>
+                    <InputGroupAddon addonType="prepend">
+                      <InputGroupText>@Security</InputGroupText>
+                    </InputGroupAddon>
+                    <Input />
+                  </InputGroup>
+                  <br />
+                </CardBody>
+                <div className="d-flex justify-content-center pt-2">
+                  <Button color="primary" size="md" className="mr-1 btn-block mt-0" onClick={() => { this.props.history.push('/home'); }} >Save</Button>
```

Comment:
I'm guessing these buttons don't arent finished with their functionality?

Reply:
Should add 2 function that handles the saving and going back respectively. You can add the navigation inside those functions and leave a TODO comment if you plan on adding the functionality later.